.github/workflows/linux.yml

```diff
@@ -1008,11 +1008,16 @@ jobs:
 
       - name: Python ONNX operators tests
         run: |
-          # Skip test_onnx/test_zoo_models and test_onnx/test_backend due to long execution time - ONNX Model Zoo tests are run separately
+          # Skip test_onnx/test_zoo_models due to long execution time - ONNX Model Zoo tests are run separately
           python3 -m pytest -sv ${OPENVINO_REPO}/src/frontends/onnx/tests -k 'not cuda' \
             --junitxml=${INSTALL_TEST_DIR}/TEST-onnx_frontend.xml \
             --ignore=${OPENVINO_REPO}/src/frontends/onnx/tests/test_python/test_zoo_models.py
 
+      - name: nGraph Tests
+        run: |
+          python3 -m pytest -v ${OPENVINO_REPO}/src/bindings/python/tests_compatibility/test_ngraph -k 'not _cuda' \
+            --junitxml=${INSTALL_TEST_DIR}/TEST-ngraph.xml
```

Comment:
how is it different from Python API 1.0 tests?

Reply:
Seems like it is no different 🥲. So I suppose the Linux nGraph ONNX pipeline could be deleted if there are no more missing parts from there in the Linux GHA pipeline.

Reply:
Linux ONNX tests on ONNX model zoo, which we don't have in GHA I suppose

Reply:
But we cannot have them as:
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/dbbdcd3a-483a-445a-b5a1-cefcbb6e3e7f)